docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md

```diff
@@ -0,0 +1,88 @@
+# GROUP BY
+
+The `GROUP BY` clause groups the rows with the same value into summary rows. Then operations such as counting, sorting, and calculation can be applied.
+
+`GROUP BY` works after the pipe symbol and before a `YIELD` or `RETURN` clause.
+
+## Group and count the output
+
+The following statement finds all the vertices connected directly to vertex `"player100"`, groups the result set by player names, and counts the times that the names show up in the result set.
+
+```ngql
+nebula> GO FROM "player100" \
+        OVER follow BIDIRECT \
+        YIELD $$.player.name as Name | \
+        GROUP BY $-.Name \
+        YIELD $-.Name as Player, COUNT(*) AS Name_Count;
++---------------------+------------+
+| Player              | Name_Count |
++---------------------+------------+
+| "Tiago Splitter"    | 1          |
++---------------------+------------+
+| "Aron Baynes"       | 1          |
++---------------------+------------+
+| "Boris Diaw"        | 1          |
++---------------------+------------+
+| "Manu Ginobili"     | 2          |
++---------------------+------------+
+| "Dejounte Murray"   | 1          |
++---------------------+------------+
+| "Danny Green"       | 1          |
++---------------------+------------+
+| "Tony Parker"       | 2          |
++---------------------+------------+
+| "Shaquille O'Neal"   | 1          |
```

Comment:
why this line not aligned? Could u pls adjust it manually?

Reply:
Good catch!